test/powershell/Modules/Microsoft.PowerShell.Utility/Send-MailMessage.Tests.ps1

```diff
@@ -181,4 +181,23 @@ Describe "Basic Send-MailMessage tests" -Tags CI {
         $mail.Body.Count | Should -BeExactly 1
         $mail.Body[0] | Should -BeExactly $body
     }
+
+    $ItArgs = $PesterArgs.Clone()
+    $ItArgs['Name'] = "Can send mail message from user to self without subject " + $ItArgs['Name']
+
+    It @ItArgs {
+        $body = "Greetings from me."
+        $subject = "Test message"
```

Comment:
Please remove the line.

Reply:
Good catch I took out the subject line from the test. Thank you :).